src/Cthulhu.jl

```diff
@@ -169,6 +169,10 @@ function _descend(mi::MethodInstance; iswarn::Bool, params=current_params(), opt
 
             if callsite.info isa MultiCallInfo
                 sub_callsites = map(ci->Callsite(callsite.id, ci), callsite.info.callinfos)
+                if isempty(sub_callsites)
+                    @warn "Found MultiCallInfo, but sub callinfos is empty"
```

Comment:
this tells me *nothing* as a user.
Should it also tell some other things like "plz make bug report?"

Reply:
IMO "plz make a bug report" is always implied when you receive a warning from code that you didn't write...